tests/jeos/openstack_check_boot.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright 2019 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: JeOS OpenStack image validation
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+    my $reg_code = get_var('SCC_REGCODE');
+
+    $self->select_serial_terminal();
+
+    my $provider = $self->provider_factory();
+    my $instance = $provider->create_instance();
+
+    record_info('Uname', $instance->run_ssh_command(cmd => q(uname -a)));
```

Comment:
Don't you need to wait for the instance to boot up? :smile:

Reply:
no, this is done already by our framework. 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/publiccloud/provider.pm#L260
and
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/publiccloud/instance.pm#L231